app/app.go

```diff
@@ -395,10 +412,11 @@ func NewOsmosisApp(
 	// register the proposal types
 	govRouter := govtypes.NewRouter()
 	govRouter.AddRoute(govtypes.RouterKey, govtypes.ProposalHandler).
-		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
+		AddRoute(paramstypes.QuerierRoute, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
```

Comment:
TODO: Double check this

Reply:
Don't see why this is swapped to QuerierRoute.  Should still just use RouterKey

See upstream:  https://github.com/cosmos/cosmos-sdk/blob/master/simapp/app.go#L293

Reply:
Undid that, wasn't clear to me either on reviewing the changes